projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts

```diff
@@ -45,7 +46,7 @@ export class AngularPlugin extends BaseTelemetryPlugin {
     public priority = 186;
     public identifier = "AngularPlugin";
 
-    constructor() {
+    constructor(private _injector: Injector = null) { // _injector is optional to provide
```

Comment:
What about defining this as optional with no default value (as specifying a default causes TypeScript to emit additional code which we don't really need.

`constructor(private _injector?: Injector)`